### PR TITLE
Bug 1848987: Add hint to switch username

### PR DESCRIPTION
**Pull request by gekorob**
Add hint message to notify the user that with Ctrl+C is possible to
switch the username already choosen

Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=1848987

### CODE REVIEW

File: pkg/asset/installconfig/ovirt/credentials.go [hunk 1]
```diff
@@ -97,12 +97,18 @@ func (c *clientHTTP) checkURLResponse() error {
 // askPassword will ask the password to connect to Engine API.
 // The password provided will be added in the Config struct.
 // If an error happens, it will ask again username for users.
-func askPassword(c *Config) error {
+func askPassword(c *Config, hintMsg string) error {
```
Comment (dougsland) [inline pkg/asset/installconfig/ovirt/credentials.go:100]: I believe we don't need hintMsg as param for a askPassword() function.

Comment (gekorob) [reply pkg/asset/installconfig/ovirt/credentials.go:100]: No pb I'll remove it

**Comment by gekorob** [comment]
Prompt for password should look like below now:

pwd_hint

**Comment by dougsland** [comment]
I have also verified the patch.

**Pull request merged**